content/en/docs/about/community.md

```diff
@@ -137,14 +154,19 @@ The following table outlines which components are maintained by each Working Gro
         </td>
       </tr>
       <!-- ======================= -->
-      <!-- Deployment Working Group -->
+      <!-- Data Working Group -->
       <!-- ======================= -->
       <tr>
-        <td rowspan="1" class="align-middle">
-          <a href="https://github.com/kubeflow/community/tree/master/wg-deployment">Deployment</a>
+        <td rowspan="2" class="align-middle">
+          <a href="https://github.com/kubeflow/community/tree/master/wg-data">Data</a>
```

Comment:
This depends on another PR https://github.com/kubeflow/community/pull/673

Reply:
@terrytangyuan Do we want to remove this part from this PR for now ?

Reply:
I removed it for now.